dashboards/etcd/etcd.json

```diff
@@ -1,566 +1,517 @@
-              "platformOptions": {
-                "ignoreTimeRange": false
-              },
-              "yAxisLeft": {
-                "zero": true
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.table"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT filter(latest(server_version), where metricName = 'etcd_server_version') as 'Etcd Server Version', if((max(etcd_server_has_leader) * cardinality(etcd_server_has_leader)) = 1, 'True', 'False') as 'Has Leader' FROM Metric where instrumentation.name = 'remote-write' and prometheus_server IN ( {{ prometheus_server }} ) and instance IN ( {{ instance }} ) facet prometheus_server SINCE 2 minutes ago until now"
               }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": true
             }
+          }
+        },
+        {
+          "title": "Process Resident Memory (MB)",
+          "layout": {
+            "column": 8,
+            "row": 4,
+            "width": 5,
+            "height": 3
           },
-          {
-            "title": "Server Proposals Applied (ops/sec)",
-            "layout": {
-              "column": 1,
-              "row": 7,
-              "width": 4,
-              "height": 3
-            },
-            "linkedEntityGuids": null,
-            "visualization": {
-              "id": "viz.line"
-            },
-            "rawConfiguration": {
-              "facet": {
-                "showOtherSeries": false
-              },
-              "legend": {
-                "enabled": true
-              },
-              "nrqlQueries": [
-                {
-                  "accountId": 0,
-                  "query": "SELECT rate(sum(etcd_server_proposals_applied_total), 1 SECONDS) FROM Metric SINCE 60 MINUTES AGO UNTIL NOW FACET instance LIMIT 100 TIMESERIES 300000 SLIDE BY 60000"
-                }
-              ],
-              "platformOptions": {
-                "ignoreTimeRange": false
-              },
-              "yAxisLeft": {
-                "zero": true
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.bar"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT latest(process_resident_memory_bytes) / 1e6 as 'Memory (MB)' FROM Metric where instrumentation.name = 'remote-write' and prometheus_server IN ( {{ prometheus_server }} ) and instance IN ( {{ instance }} ) and (app_kubernetes_io_name = 'etcd' or app_newrelic_io_name = 'etcd' or k8s_app = 'etcd' or job like '%etcd%' or instance like '%:2381') FACET prometheus_server, instance LIMIT 100"
```

Comment:
Should we change the labels condition from <k8slabel>='etcd' to <k8slabel>l like '%etcd%' 
I'm [seeing etcd pods](https://docs.google.com/spreadsheets/d/1Rtp6FUgTbP49MKT8RFp1nuBftlRxKdLLQWbNBAyz1xg/edit#gid=1831540668) with labels like 'etcd-manager-events' or 'etcd-manager-main'